fivox/uriHandler.cpp

```diff
@@ -495,11 +498,18 @@ ImageSourcePtr< TImage > URIHandler::newImageSource() const
         source = EventValueSummationImageSource< TImage >::New();
         break;
     default:
-        auto functorSource = FunctorImageSource< TImage >::New();
-        auto functor = newFunctor< TImage >();
-        functorSource->setFunctor( functor );
-        functor->setEventSource( eventSource );
-        source = functorSource;
+#ifdef USE_CUDA
```

Comment:
FIVOX_USE_CUDA?
On the cluster it means it will always use cuda. A runtime option to choose between cuda, CPU and maybe ispc could be needed depending on performance differences with different datasets and/or memory constrains.